liquibase-snowflake/src/main/java/liquibase/diff/output/changelog/core/ChangedSequenceChangeGeneratorSnowflake.java

```diff
@@ -0,0 +1,47 @@
+package liquibase.diff.output.changelog.core;
+
+import liquibase.change.Change;
+import liquibase.change.core.AlterSequenceChange;
+import liquibase.database.Database;
+import liquibase.database.core.SnowflakeDatabase;
+import liquibase.diff.ObjectDifferences;
+import liquibase.diff.output.DiffOutputControl;
+import liquibase.diff.output.changelog.ChangeGeneratorChain;
+import liquibase.structure.DatabaseObject;
+import liquibase.structure.core.Sequence;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ChangedSequenceChangeGeneratorSnowflake extends ChangedSequenceChangeGenerator{
```

Comment:
Just wondering if the name of the class should be be SnowflakeChangedSequenceChangeGenerator? Can you please check if there is a pattern, if not am fine..

Reply:
The naming here goes according to the rest of the classes in Snowflake extension, so I'd consider it to be correct